emData/download.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# fw_synch_200515
+tarball_url="https://cernbox.cern.ch/index.php/s/CipX7CfTXIj1lcK/download"
```

Comment:
looks like the tarball_url variable isn't used in the `wget`

Reply:
Doh! Just fixed it.